react-native-libraries.json

```
@@ -12202,5 +12202,16 @@
     "examples": ["https://github.com/nativeflowteam/nativeflowcss/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/grahammendick/navigation",
+    "npmPkg": "https://www.npmjs.com/package/navigation",
```

Hello @gerzonc, thanks for adding the new entry!

There are two thing that need to corrected, before we can ship that addition - `npkPkg` field should match the package name, and the `githubUrl`, in monorepo, need to point directly to the directory when a `package.json` file for a given package resides.

```suggestion
    "githubUrl": "https://github.com/grahammendick/navigation/blob/master/build/npm/navigation",
```

Btw. The repository structure is quite unusual, those there might be some other minor problems, but we will try to address them! It might be a good idea to also add other packages from monorepo, like `navigation-react` and `navigation-react-native`, if they are required when using `navigation` lib in React Native apps.

---

Aah, I see, interesting, I'll take a look into fixing those issues, thanks!